test/support/pages/DiscoverPage.js

```diff
@@ -101,8 +107,30 @@ define(function (require) {
       .then(function (bars) {
         return bars;
       });
+    },
+
+    getChartInterval: function getChartInterval() {
+      return thisTime
+      .findByCssSelector('span.results-interval:nth-child(2) > a:nth-child(1)')
+      .getVisibleText();
+    },
+
+    setChartInterval: function setChartInterval(interval) {
+      return thisTime
+      .findByCssSelector('span.results-interval:nth-child(2) > a:nth-child(1)')
+      .click()
+      .catch(function () {
+        // in some cases we have the link above, but after we've made a
+        // selection we just have a select list.
+      })
+      .then(function () {
+        return thisTime
```

Comment:
can we use [end](http://theintern.github.io/leadfoot/Command.html#end) instead of nesting?